pkg/planner/core/point_get_plan.go

```diff
@@ -1157,6 +1160,64 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt, check bool
 	return nil
 }
 
+func checkGlobalIndexForPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt, pairs []nameValuePair, check bool) *PointGetPlan {
+	tblName, tblAlias := getSingleTableNameAndAlias(selStmt.From)
```

Comment:
Cannot these be provided as arguments, since they are done already in the calling function?

Reply:
I considered these be provideed as arguments, but the disadvantage is arguments are too many. I don't think too many arguments is good. 
@mjonss , what's your advice？If you think these arguments should not be done secondly in this PR, I will refactor this.